sound/soc/codecs/rt5682.c

```diff
@@ -2935,6 +2935,9 @@ static int rt5682_set_sdw_stream(struct snd_soc_dai *dai, void *sdw_stream,
 {
 	struct sdw_stream_data *stream;
 
+	if (!sdw_stream)
+		return 0;
+
```

Comment:
@plbossart   if (!sdw_stream)  we need to clear dma_data in dai at line L2948.

Reply:
it's already done in the shutdown part.

Reply:
yes, it is done in cdns_set_sdw_stream.